benchmark/benchmark.py

```diff
@@ -21,14 +21,23 @@ def _path(x):
     parser.add_argument('--noise', required=True)
 
     args = parser.parse_args()
+    args_dict = vars(args)
 
     if args.gcp_credential_path is not None:
         os.environ['GOOGLE_APPLICATION_CREDENTIALS'] = args.gcp_credential_path
 
     if args.ibm_credential_path is not None:
-        os.environ['IBM_CREDENTIALS_FILE'] = args.ibm_credential_path
-
-    engine = NLUEngine.create(NLUEngines[args.engine_type], args.gcp_project_id, args.ibm_model_id, args.ibm_custom_id)
+        with open(args.ibm_credential_path) as f:
+            for line in f:
+                if not line.strip():

```

Comment:
are you checking if the line is empty? If yes, please do it explicitly:

if len(line.strip()) > 0:
...